packages/web-components/src/components/rux-global-status-bar/test/index.html

```diff
@@ -17,8 +17,76 @@
         <script nomodule src="/build/astro-web-components.js"></script>
 
         <link rel="stylesheet" href="/build/astro-web-components.css" />
+        <script
+            type="module"
```

Comment:
Mark had me comment out the script and holster. Apparently it affects Cypress.

Reply:
Thank you! I've been punching cypress in the face but it wasn't helping.